jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -128,6 +128,10 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmp_path):
     assert len(dc.links) == 22
     assert len(dc.external_links) == 4  # pixel linked
     # Link 3D z to 2D x and 3D y to 2D y
+    assert (dc.external_links[0].cids1[0].label == "Pixel Axis 0 [z]" and
```

Comment:
I confirmed locally that this test fails on `main` without this patch as follows:

```
E       AssertionError: assert ('Pixel Axis 0 [z]' == 'Pixel Axis 0 [z]'
E
E           Pixel Axis 0 [z] and 'Pixel Axis 0 [y]' == 'Pixel Axis 1 [x]'
E
E         - Pixel Axis 1 [x]
E         ?            ^  ^
E         + Pixel Axis 0 [y]
E         ?            ^  ^)
```